server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/MailboxGet.scala

```diff
@@ -0,0 +1,40 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.mail
+
+import eu.timepit.refined.types.string.NonEmptyString
+import org.apache.james.jmap.model.AccountId
+import org.apache.james.jmap.model.State.State
+import org.apache.james.mailbox.model.MailboxId
+
+case class Ids(value: List[MailboxId])
+
+case class Properties(value: List[NonEmptyString])
+
+case class MailboxGetRequest(accountId: AccountId,
+                             ids: Ids,
```

Comment:
is empty list allowed?

Reply:
From the specs:
```
ids: Id[]|null The ids of the Foo objects to return. If null, then all records of the data type are returned, if this is supported for that data type and the number of records does not exceed the maxObjectsInGet limit.
```

They don't really say about an empty list, so I guess it is allowed..? I'm not sure if we should treat it as the null case or as if there is a list but it's empty, then we don't return any mailboxes

WDYT?

Reply:
Ah no wait... A bit further in the specs, for the `list` argument in response:

> list: Foo[] An array of the Foo objects requested. This is the empty array if no objects were found or if the ids argument passed in was also an empty array. The results MAY be in a different order to the ids in the request arguments. If an identical id is included more than once in the request, the server MUST only include it once in either the list or the notFound argument of the response.

So if the `ids` argument is an empty array, then we don't return any mailboxes